README.md

```diff
@@ -13,9 +13,8 @@ Bedrock helps you:
 * [Walkthrough: Deploying a First Workload](./docs/firstWorkload)
 * [Deep Dive: Why GitOps?](./docs/why-gitops.md)
 
-## Infrastructure Management
-* [Walkthrough: Deploying and Maintaining Infrastructure](./docs/infrastructure.md)
-* [CLI Reference](https://github.com/CatalystCode/spk/blob/master/guides/cloud-infra-management.md)
+## First Cluster
```

Comment:
I would recommend just keeping the two cluster pieces together.  The day two part I'll just cover in the deep dive section.